react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+let intervalId;
```

Comment:
store the `intervalId` on `this` so that you can have more than one instance of your carousel without their intervals clashing.